### ISSUE TITLE
The currentColClassName parameter's missing

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/docs#60]
The `currentColClassName` option is supposed to accept a single `string` as a value (it's described this way in the docs and in the definition file) but it also accepts an array of `strings` and executes the styling added with all the classes in the array.

So if I'm right, we should replace `currentColClassName?: string;` with `currentColClassName?: string | string[];`

**Demo** 
https://jsfiddle.net/handsoncode/fvqjxbhL/1/

**Settings**
> currentColClassName: ['currentColumn', 'abc']

**Result**
![image](https://user-images.githubusercontent.com/10757813/61875339-b5f7b180-aeea-11e9-904a-ffb684b956fb.png)

**Comment by AMBudnik** [comment]
This rule also applies to

-  `commentedCellClassName`, 
- `currentRowClassName`,
- `tableClassName` (which is a bit more immune for styling), but also is correctly described with the definition file and in the documentation (as the only one from the list) `tableClassName?: string | string[];`

It doesn't relate to 
- `currentHeaderClassName`, 
- `invalidCellClassName`, 
- `noWordWrapClassName`, 
- `placeholderCellClassName`, 
- `readOnlyCellClassName`.

**Comment by AMBudnik** [comment]
It also applies to v8.0.0 https://jsfiddle.net/uh7gx264/

![ok](https://user-images.githubusercontent.com/10757813/91045217-d676cb80-e616-11ea-9861-a666c2c0c4e3.gif)

You just need to select any cell to see that both classes (array of string) are executed.

ps. It looks like this scenario (more than one class) has been predicted by this test https://github.com/handsontable/handsontable/blob/638c3aa0303af292b3e685046f53e6e9ea17208c/test/e2e/settings/currentRowClassName.spec.js#L61

**Comment by wojciechczerniak** [comment]
> ps. It looks like this scenario (more than one class) has been predicted by this test https://github.com/handsontable/handsontable/blob/638c3aa0303af292b3e685046f53e6e9ea17208c/test/e2e/settings/currentRowClassName.spec.js#L61

This tests (and all in the file) sets just a single class. 

https://github.com/handsontable/handsontable/blob/638c3aa0303af292b3e685046f53e6e9ea17208c/test/e2e/settings/currentRowClassName.spec.js#L55

The line you're refering to is checking the length of an array that should contain TD elements with the class. Not related to how we handle classes internally.

**Comment by wojciechczerniak** [comment]
Our `addClass` helper that is used internally supports a string (`"class1"`), string with multiple classes (`"class1 class2"`) or an array (`["class1", "class2"]`):

https://github.com/handsontable/handsontable/blob/638c3aa0303af292b3e685046f53e6e9ea17208c/src/helpers/dom/element.js#L353

Now, the difference is because we sometimes use this helper as is and an array will be handled in addition to strings, and sometimes (baseRenderer mainly) we do some operations with the classes before the actual `addClass` call:

https://github.com/handsontable/handsontable/blob/638c3aa0303af292b3e685046f53e6e9ea17208c/src/renderers/_cellDecorator.js#L7-L38

TL:DR; I haven't seen a test that would would confirm that our spec allows an array for those options. Therefore strings with single className are the only allowed values. You can create a feature request to accept arrays of classes. This will require refactoring base renderer and adding tests for all className options so this will not change in the future. Until then an array is not guaranted.

**Comment by AMBudnik** [comment]
Thank you for the explanation @wojciechczerniak
